src/lib/operations/create-bundle-and-sign/create-bundle-and-sign.handler.ts

```diff
@@ -0,0 +1,55 @@
+import express from "express";
+import { ExpressRequestWithToken } from "../../utils/auth/auth0";
+import { createBundleAndSign } from "./createBundleAndSign";
+import { Route } from "../../server/server.constants";
+import { logRequestSuccess, logRequestStart } from "../../utils/log/log.utils";
+
+export interface CreateBundleAndSignIdTokenData {
+  data: string;
+  keyName: string;
+  owner: string;
+  tags: string[];
+}
+
+export function createBundleAndSignHandlerFactory() {
+  return async (
+    req: ExpressRequestWithToken<CreateBundleAndSignIdTokenData>,
+    res: express.Response,
+  ) => {
+    try {
+      const { idToken } = req;
+      const { data } = idToken;
+
+      // TODO: Replace with Joi.
+      if (
```

Comment:
Once I make the changes to not include files/attachments directly in the token (in a new PR), I'll also add Joi schemas to validate the shape of the rest of the data being sent.